server/etcdmain/config.go

```diff
@@ -160,6 +160,7 @@ func newConfig() *config {
 	fs.DurationVar(&cfg.ec.GRPCKeepAliveMinTime, "grpc-keepalive-min-time", cfg.ec.GRPCKeepAliveMinTime, "Minimum interval duration that a client should wait before pinging server.")
 	fs.DurationVar(&cfg.ec.GRPCKeepAliveInterval, "grpc-keepalive-interval", cfg.ec.GRPCKeepAliveInterval, "Frequency duration of server-to-client ping to check if a connection is alive (0 to disable).")
 	fs.DurationVar(&cfg.ec.GRPCKeepAliveTimeout, "grpc-keepalive-timeout", cfg.ec.GRPCKeepAliveTimeout, "Additional duration of wait before closing a non-responsive connection (0 to disable).")
+	fs.StringVar(&cfg.ec.ReadOnlyOption, "read-only-option", cfg.ec.ReadOnlyOption, "Option to support the linearizability of the read only request(ReadOnlySafe and ReadOnlyLeaseBased are supported options).")
```

Comment:
Could you, please reference in description: 
  https://godoc.org/github.com/coreos/etcd/raft#ReadOnlyOption

Reply:
Sorry, I don't find any description of `ReadOnlyOption` in the doc, maybe the description is removed?

Reply:
The corrected link (thanks to 3.5.0-alpha.0 release): 
  https://pkg.go.dev/go.etcd.io/etcd/raft/v3#ReadOnlyOption